### ISSUE TITLE
Cannot Use http.WrapClient() with Go 1.19

### ISSUE THREAD

**Issue by evanlurvey** [DataDog/dd-trace-go#1455]
When trying to wrap a HTTP client in go 1.19 I get this panic 

```
panic: Something in this program imports go4.org/unsafe/assume-no-moving-gc to declare that it assumes a non-moving garbage collector, but your version of go4.org/unsafe/assume-no-moving-gc hasn't been updated to assert that it's safe against the go1.19 runtime. If you want to risk it, run with environment variable ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.19 set. Notably, if go1.19 adds a moving garbage collector, this program is unsafe to use.

goroutine 1 [running]:
go4.org/unsafe/assume-no-moving-gc.init.0()
        /Users/elurvey/go/pkg/mod/go4.org/unsafe/assume-no-moving-gc@v0.0.0-20211027215541-db492cf91b37/untested.go:25 +0x1f4
exit status 2
```

Souce Code
```go
package main

import (
	ddhttp "gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http"
	"net/http"
)

func main() {
	ddhttp.WrapClient(&http.Client{})
}
```

**Comment by nsrip-dd** [comment]
This is fixed in dd-trace-go version v1.41.0.

**Comment by evanlurvey** [comment]
Ah that’s my bad. I should have checked version
